jda-integration/src/main/java/me/koply/kcommando/integration/impl/jda/JDAIntegration.java

```diff
@@ -89,7 +89,7 @@ public void registerSlashCommand(SlashBox box) {
             Guild guild = api.getGuildById(guildId);
             if (guild != null) {
                 guild.upsertCommand(data).queue();
-            } if (KCommando.verbose) {
+            } else if (KCommando.verbose) {
```

Comment:
what?

Reply:
I thought you forgot to put else for readable code conditions must start on a new line.